src/main/resources/db/migration/V4__create_project_table.sql

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
This will need to be V5, as there already is a V4 that exists. Remember to test your migrations after you pull in from the main branch :)

Reply:
Just check main and only saw V3 as the highest. Where is the V4?

Reply:
<img width="1036" alt="Screenshot 2024-06-27 at 14 22 05" src="https://github.com/ThomasS-kainos/GitHelp-dropwizard-flyway-starter/assets/118667330/1c5f6785-71ba-4153-800a-8ea918506c82">

Reply:
Whoops! Oversight by me, approving now ..